src/intrinsic/mod.rs

```diff
@@ -886,20 +858,43 @@ impl<'a, 'gcc, 'tcx> Builder<'a, 'gcc, 'tcx> {
     }
 
     fn count_leading_zeroes(&mut self, width: u64, arg: RValue<'gcc>) -> RValue<'gcc> {
+        let func = self.current_func();
```

Comment:
Please add a comment to this method to describe what it does (return 0 if arg is zero, otherwise call the leading_zero builtin).